src/clinical/api/clinical-api.ts

```diff
@@ -36,7 +36,10 @@ import {
   ClinicalErrorsApiBody,
   ClinicalSearchApiBody,
   CompletionState,
+  DonorDataApiBody,
 } from './types';
+import { loggerFor } from '../../logger';
+const L = loggerFor(__filename);
```

Comment:
Could `Log`, or `Logger` do the same job, while being a bit more declarative than a single capital letter `l`?

Reply:
yes and it would be better. I'm being consistent with the existing pattern in this repo. I would - recommend a separate action to rename them all.